.github/workflows/format.yml

```diff
@@ -0,0 +1,56 @@
+name: Format check
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.py'
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    outputs:
+      files: ${{ steps.git-diff-filter.outputs.files }}
+    name: Detect added and changed files
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: GrantBirki/git-diff-action@v2
+        id: git-diff-action
```

Comment:
Can you make this name (slightly) different from the action name? Right now it's confusing to tell which object is being referred to.

Reply:
Updated